Sources/Datadog/RUMMonitor.swift

```diff
@@ -0,0 +1,35 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+public class RUMMonitor {
+    private let rumApplicationID: String
+
+    public init(rumApplicationID: String) {
+        self.rumApplicationID = rumApplicationID
+    }
+
+    /// TODO: RUMM-585 Replace with real RUMMonitor public API
+    public func sendFakeViewEvent(viewURL: String) {
```

Comment:
can't we `@testable import Datadog` in `Example` app instead of having this public method in `master` branch?

Reply:
This won't be merged to `master`.

`@testable import` is disabled in Example app's project - it hosts integration tests, and we want those tests to only use public APIs. IMO exposing one public method on a `rum` feature branch is better & easier than changing the project configuration for debugging purpose. Please also note - it's clearly marked with `TODO: RUMM-585` and will never get into `master` before it gets resolved.